submissions/tavor118sn/task_js_post_oop/index.js

```diff
@@ -0,0 +1,90 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: _put repo URL here_
+   Web app: _put project's github pages URL here_
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+
+
+class Creature {
+  constructor(species, name, age, gender, legs, say, friends) {
+    this.species = species;
+    this.name = name;
+    this.age = age;
+    this.gender = gender;
+    this.legs = legs;
+    this.saying = say;
+    this._friends = friends
+  }
+
+  get friends() {
+    let friendsRepresentation = '';
+    if (this._friends && this._friends.length > 0) {
+      friendsRepresentation = this._friends.join(', ');
+    }
+    return friendsRepresentation;
+  }
+
+  getAllCharacteristics() {
+    return [
+      `${this.species}`,
+      `<strong>${this.name}</strong>`,
+      `${this.age}`,
+      `${this.gender}`,
+      `${this.legs}`,
+      `${this.saying}`,
+      `${this.friends}`,
+    ]
+  }
+
+  toString() {
+    return this.getAllCharacteristics().join('; ')
+  }
+}
+
+class Human extends Creature {
+  constructor(name, age, gender, saying, friends) {
+    let species = 'human';
+    let legs = 2;
```

Comment:
Try to implement with [default parameters](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters)